contracts/DCNT721A.sol

```diff
@@ -96,13 +127,81 @@ contract DCNT721A is ERC721A, Initializable, Ownable, Splits {
     }
 
     _safeMint(msg.sender, numberOfTokens);
-    for (uint256 i = 0; i < numberOfTokens; i++) {
-      emit Minted(msg.sender, mintIndex++);
+    unchecked {
+      for (uint256 i = 0; i < numberOfTokens; i++) {
+        emit Minted(msg.sender, mintIndex++);
+      }
+    }
+  }
+
+  // allows the owner to "airdrop" users an NFT
+  function mintAirdrop(address[] calldata recipients) external onlyOwner {
+    uint256 atId = _nextTokenId();
+    uint256 startAt = atId;
+    require(atId + recipients.length <= MAX_TOKENS,
```

Comment:
`<=` costs more gas than `<` + ` +1`

```suggestion
    require(atId + recipients.length < MAX_TOKENS + 1,
```

source: https://nftchance.medium.com/the-gas-efficient-way-of-building-and-launching-an-erc721-nft-project-for-2022-b3b1dac5f2e1